CONTRIBUTING.md

```diff
@@ -1,75 +1,47 @@
 
-To navigate our codebase with confidence, see [Reflex Docs](https://reflex.dev/docs/getting-started/introduction) :confetti_ball:.
 
-### Discussions
 
-- Have a question? Want to discuss a feature? [Start a discussion](https://github.com/pynecone-io/pynecone/discussions)
-
-    We welcome any discussions and questions. We want to make sure that Reflex is the best it can be, and we can't do that without your help.
-
-### Issues
-
-* #### Create a new issue
-
-    If you spot a problem with anything in Reflex feel free to create an issue. Even if you are not sure if its a problem with the framework or your own code, create an issue and we will do our best to answer or resolve it.
-
-* #### Solve an issue
-
-    Scan through our [existing issues](https://github.com/pynecone-io/pynecone/issues) to find one that interests you. You can narrow down the search using `labels` as filters. As a general rule, we don’t assign issues to anyone. If you find an issue to work on, you are welcome to open a PR with a fix. Any large issue changing the compiler of Reflex should brought to the Reflex maintainers for approval
+## Runing a Local Build of Reflex 
+Here is a quick guide to how the run Reflex repo locally so you can start contributing to the project.
 
-Thank you for supporting Reflex!🎊
+**Pre-requisites:**
+- Python >= 3.7
+- Poetry version >= 1.4.0 and add it to your path (see [Poetry Docs](https://python-poetry.org/docs/#installation) for more info).
 
-## 💻 How to Run a Local Build of Reflex 
-Here is a quick guide to how the run Reflex repo locally so you can start contributing to the project.
 
-First clone Reflex:
+**1.  Clone Reflex and navigate into the repo:**
 ``` bash
 git clone https://github.com/reflex-dev/reflex.git
-```
-
-Navigate into the repo:
-``` bash
 cd reflex
 ```
 
-Install poetry version >= 1.4.0 and add it to your path (see [Poetry Docs](https://python-poetry.org/docs/#installation) for more info).
-
-Install your local Reflex build:
+**2. Install your local Reflex build:**
 ``` bash
 poetry install
 ```
-
-Now create an examples folder so you can test the local Python build in this repository:
+**3. Now create an examples folder so you can test the local Python build in this repository.**
+* We have the examples folder in the .gitignore, so your changes in reflex/examples won't be reflected in your commit.
 ``` bash
 mkdir examples
 cd examples
 ```
 
-Create a project in this folder can be named anything but for the sake of the directions we'll use `example`:
-``` bash
-mkdir example
-cd example
-```
-
-Now Init/Run
+**4. Init and Run**
 ``` bash
 poetry run reflex init
 poetry run reflex run
 ```
-
 All the changes you make to the repository will be reflected in your running app.
-* We have the examples folder in the .gitignore, so your changes in reflex/examples won't be reflected in your commit.
 
-## 🧪 Testing and QA
 
-Any feature or significant change added should be accompanied with unit tests.
```

Comment:
Why did we remove this? Is it because it's repeated below?

Reply:
Yes